ext_emconf.php

```diff
@@ -10,10 +10,10 @@
     'author' => 'Patrick Broens, Ralf Zimmermann',
     'author_email' => 'patrick@patrickbroens.nl, ralf.zimmermann@tritum.de',
     'author_company' => '',
-    'version' => '8.7.0',
+    'version' => '9.5.0',
```

Comment:
we'll go with 9.0.0 then, as the next major release will be semver focussed, I'd suggest.

Reply:
fine